src/Marquee.tsx

```diff
@@ -106,6 +105,7 @@ export class Marquee extends Component<Props, State> {
                 <div
                     className={`preact-marquee__content ${this.state.animationClassName} ${this.state.pauseWhenHoveredClassName}`}
                     ref={this.saveContentReference}
```

Comment:
I think these ref functions are unnecessary. They will not be called anywhere else and they are not doing any additional things.

Reply:
it's just a callback to save the reference. what is the alternative?

Reply:
Just using the prop itself like `this.marqueeElement`... For now I do not see any benefit but we can also keep it.